### PR TITLE
Comments for sxhkdrc

### PR DESCRIPTION
**Pull request by Krak-n**
As follow through on issue #244, I have commented most of the default sxhkdrc commands. I am still a bit lost on a few of the commands. I'm hoping others have a bit more of a grasp over the commands and are willing to contribute so this can be fully merged.

### CODE REVIEW

File: examples/sxhkdrc [hunk 1]
```diff
@@ -2,74 +2,97 @@
...
 
+# Set the split ratio of the selected window.
 super + ctrl + {1-9}
 	bspc window -r 0.{1-9}
 
+# Send seleced window to a new desktop???
 super + {_,shift + }{1-9,0}
 	bspc {desktop -f,window -d} ^{1-9,10}
-
+
```
Comment (Stebalien) [inline examples/sxhkdrc]: Space.

File: examples/sxhkdrc [hunk 2]
```diff
@@ -77,9 +100,11 @@ super + @button{1-3}
 # wm independent hotkeys
 #
 
+# Start a terminal (URXVT is the default for the example)
 super + Return
 	urxvt
-
+
```
Comment (Stebalien) [inline examples/sxhkdrc]: Space.

**Comment by Krak-n** [comment]
Closing for breaking a belief of the project.

**Comment by svenstaro** [comment]
I think that actually these are quite good for getting started. Some are rather obvious, yes. However, others aren't and personally I also put comments like these in my sxhkdrc since sometimes I forgot and it makes it easy to look up but why I want to accomplish instead of having to study the manpage every time.

**Comment by r8b7xy** [comment]
This was against the concpet of bspwm as a bloat free and clean wm imho. Just have a look at sxhkdrc deafult config - nice and clean. There's no point to add those comments as most of the bspwm recipient are rather experienced powerusers.

**Pull request closed without merging**